### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by V0XNIHILI**
Fix a few things that made it impossible to setup a cluster from scratch with the current commit on main.

### CODE REVIEW

File: terraform/terraform-dependencies/main.tf [hunk 1]
```diff
@@ -15,7 +15,7 @@ resource "kustomization_resource" "training_operator" {
 # Create NFS resource
 resource "helm_release" "nfs_client_provisioner" {
   name       = var.nfs_provider_information.release_name
-  chart      = "charts/nfs-server-provisioner-1.1.3.tgz"
+  chart      = "${path.module}/../../charts/nfs-server-provisioner-1.3.1.tgz"
```
Comment (JMGaljaard) [inline terraform/terraform-dependencies/main.tf:18]: This seems to be related to an earlier patch that has been provided in #52. Please open an issue to the problem that you encountered to make discussion easier.

**Review by JMGaljaard — CHANGES_REQUESTED** [review]
Please address the issues raised and/or open a github issue addressing issues that you have encountered

File: jupyter/terraform_notebook.ipynb [hunk 2]
```diff
@@ -532,8 +541,22 @@
...
+    "Lastly, we need to install the dependencies on our cluster. First change the directories, and then run the `init`, `plan` and `apply` commands as we did for creating the GKE cluster."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wget https://kvaps.github.io/charts/nfs-server-provisioner-1.3.1.tgz -O ../charts/nfs-server-provisioner-1.3.1.tgz"
```
[the remark this answers is not in the document]
Comment (V0XNIHILI) [reply jupyter/terraform_notebook.ipynb:553]: Yes it is, as Helm does not do this (at least for my case and for my project group partner)

File: jupyter/terraform_notebook.ipynb [hunk 3]
```diff
@@ -312,7 +321,7 @@
     "##################\n",
     "### CHANGE ME! ###\n",
     "##################\n",
-    "OWNER_MAIL=\"mygoogleaccount@gmail.com\"\n",
+    "OWNER_MAIL=\"DouweDB@gmail.com\"\n",
```
[the remark this answers is not in the document]
Comment (V0XNIHILI) [reply jupyter/terraform_notebook.ipynb:324]: Will do, thought I didn't commit this

File: configs/example_cloud_experiment.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
    "cluster": {
       "orchestrator": {
-         "orchestrator_type": "simulated"
+         "c": "batch"
```
[the remark this answers is not in the document]
Comment (V0XNIHILI) [reply configs/example_cloud_experiment.json:4]: Sorry, copy paste mistake, will fix it.

File: terraform/terraform-dependencies/main.tf (hunk 1 above)
Comment (JMGaljaard) [reply terraform/terraform-dependencies/main.tf:18]: This has been addressed in #59

File: jupyter/terraform_notebook.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (JMGaljaard) [reply jupyter/terraform_notebook.ipynb:553]: This has now been resolved in #58

**Comment by V0XNIHILI** [comment]
I will close this PR as you fixed the main issue in #59.

**Pull request closed without merging**